python/erdos/operators/map.py

```diff
@@ -0,0 +1,17 @@
+class MapWindowOp(erdos.Operator):
+    """Operator transforms a given list of Messages using a user-defined function"""
+
+    def __init__(self, read_stream, write_stream, function):
+        read_stream.add_callback(self.callback, [write_stream])
+        self.function = function
+    
+    def callback(self, msg, write_stream):
+        """Operator expects msg.data to be a list of Messages"""
+        print("MapOp: received {msg}".format(msg=msg))
+        msg=erdos.Message(msg.timestamp, self.function(msg.data))

```

Comment:
Sorry, might've given you a bad design. Because of the way messages are used in Pylot, having the function read a message and return a message might be better. @ICGog @sukritkalra thoughts?